### PR TITLE
DAT-4051 ad related temporary changes to maps

### PR DESCRIPTION
**Pull request by RafLeszczynski**
## Description:
These are temporary changes and will be removed after the ad campaign

- Restrict editing maps that are included in ad campaign only to stuff users
- Change popup display from click to on hover for maps that are included in ad campaign

## Links:

- https://wikia-inc.atlassian.net/browse/DAT-4051
- https://wikia-inc.atlassian.net/browse/DAT-4055

DO NOT MERGE !!!!!
Need to go live together with app and config changes

### CODE REVIEW

File: assets/scripts/map.js [hunk 1]
```diff
@@ -268,7 +268,13 @@ require(
 				utils.addClass(body, 'wikia-mobile');
 				filterBox.setUpHideButton();
 			} else if (mapConfig.city_id === options.cityId) {
-				setupContributionOptions();
+				// Temporary change required for ad purpose - https://wikia-inc.atlassian.net/browse/DAT-4051
+				// We need to limit contribution options on protected maps related to the ad campaign only to stuff
+				// users.
+				// TODO: remove this as a part of https://wikia-inc.atlassian.net/browse/DAT-4055
+				if (mapConfig.isProtected !== true || options.isUserStuff) {
```
Comment (rogatty) [inline assets/scripts/map.js]: Minor - this should be `isUserStaff`

**Comment by rogatty** [comment]
LGTM

File: assets/scripts/im.poi.js [hunk 2]
```diff
@@ -70,6 +70,18 @@ define('im.poi', ['im.leafletWrapper', 'im.config', 'im.renderUI', 'im.i18n'], f
 		popup = createPoiPopup(poi, popupWidth);
 		marker.bindPopup(popup);
 
+		// Temporary change required for ad purpose - https://wikia-inc.atlassian.net/browse/DAT-4051
+		// Open popups on hover on maps related to the ad campaign
+		// TODO: remove this as a part of https://wikia-inc.atlassian.net/browse/DAT-4055
+		if (config.mapConfig.isProtected) {
+			marker.on('mouseover', function () {
+				this.openPopup();
```
Comment (rogatty) [inline assets/scripts/im.poi.js]: What will happen when user hovers on POI and then clicks it?

Comment (RafLeszczynski) [inline assets/scripts/im.poi.js]: This is still WIP and I have issues with smoke testing so I'll let you know ones I could try it out. This  PR is just an initial round of CR

File: assets/scripts/map.js (hunk 1 above)
Comment (RafLeszczynski) [inline assets/scripts/map.js]: I think `mapConfig.isProtected === false` will be easier to read

**Pull request closed without merging**